includes/class-newspack-newsletters-subscription.php

```diff
@@ -377,7 +377,7 @@ public static function add_contact( $contact, $lists = [] ) {
 		 *    @type string   $name     Contact name. Optional.
 		 *    @type string[] $metadata Contact additional metadata. Optional.
 		 * }
-		 * @param string[]      $lists    Array of list IDs to subscribe the contact to.
+		 * @param string[]|false      $lists    Array of list IDs to subscribe the contact to.
 		 * @param bool|WP_Error $result   True if the contact was added or error if failed.
```

Comment:
Nit: lint the inline doc spacing

Reply:
Hm, this was not reported by the linter. But adjusted in 202ceb7e26e53d12a281da60bfe570f224c015a6